akka-http-core/src/test/scala/akka/http/impl/model/parser/HttpHeaderSpec.scala

```diff
@@ -577,14 +585,14 @@ class HttpHeaderSpec extends FreeSpec with Matchers {
       "WWW-Authenticate: Basic realm=\"WallyWorld\"" =!=
         `WWW-Authenticate`(HttpChallenge("Basic", Some("WallyWorld")))
       "WWW-Authenticate: BaSiC rEaLm=WallyWorld" =!=
-        `WWW-Authenticate`(HttpChallenge("BaSiC", "WallyWorld")).renderedTo("BaSiC realm=\"WallyWorld\"")
+        `WWW-Authenticate`(HttpChallenge("BaSiC", Some("WallyWorld"))).renderedTo("BaSiC realm=\"WallyWorld\"")
```

Comment:
(this change isn't strictly needed anymore, but can't harm I guess)

Reply:
Yes, we don't need the `Some` now, I'll remove it.